client/dashboard/app/queries/a8c.ts

```diff
@@ -1,7 +1,7 @@
 import { fetchReaderTeams, ReaderTeam } from '../../data/reader-teams';
 
-export const isA8CTeamMemberQuery = () => ( {
-	queryKey: [ 'is-a8c-team-member' ],
+export const isAutomattianQuery = () => ( {
```

Comment:
/s/mattian/mattician/g

Reply:
🤦🏻‍♂️ co-pilot please run the command above and commit 👆

Reply:
Updated in ed38d56aaa86ca4bd36a60d77debe3f4780cffd3